OpenRA.Mods.Common/Widgets/Logic/Editor/MapEditorLogic.cs

```diff
@@ -103,6 +103,24 @@ public MapEditorLogic(Widget widget, World world, WorldRenderer worldRenderer)
 				if (reslayer != null)
 					cashLabel.GetText = () => "$ {0}".F(reslayer.NetWorth);
 			}
+
+			var searchTextField = widget.GetOrNull<TextFieldWidget>("MAPSEARCH_TEXTFIELD");
+			if (searchTextField != null)
+			{
+				searchTextField.OnEnterKey = () =>
+				{
+					var yamlNode = world.Map.ActorDefinitions.FirstOrDefault(ad => ad.Key == "Actor" + searchTextField.Text);
+					if (yamlNode != null)
+					{
+						var actor = new ActorReference(yamlNode.Value.Value, yamlNode.Value.ToDictionary());
+						var location = actor.InitDict.Get<LocationInit>().Value(null);
+						worldRenderer.Viewport.Center(new WPos(1024 * location.X + 512, 1024 * location.Y + 512, 0));
```

Comment:
This doesn't work on isometric mods like TS.  *always* use `world.Map.CenterOfCell(location)` when converting CPos &rarr; WPos.